nipype/utils/profiler.py

```diff
@@ -0,0 +1,309 @@
+# -*- coding: utf-8 -*-
+# @Author: oesteban
+# @Date:   2017-09-21 15:50:37
+# @Last Modified by:   oesteban
+# @Last Modified time: 2017-09-25 16:37:02
+"""
+Utilities to keep track of performance
+"""
+from __future__ import print_function, division, unicode_literals, absolute_import
+
+import threading
+try:
+    import psutil
+except ImportError as exc:
+    psutil = None
+
+from .. import config, logging
+from .misc import str2bool
+from builtins import open
+
+proflogger = logging.getLogger('utils')
+
+resource_monitor = str2bool(config.get('execution', 'resource_monitor'))
```

Comment:
Or raise deprecation warning here...

Reply:
get now raises warning (it is not `DeprecationWarning` bc they are filtered by default)